### ISSUE TITLE
NEP29 blog post

### ISSUE THREAD

**Issue by orbeckst** [MDAnalysis/MDAnalysis.github.io#238]
As stated in https://github.com/MDAnalysis/mdanalysis/issues/3560 we need to announce that we are following NEP29 (mostly).

@IAlibay knows best what we decided to do.

**Comment by orbeckst** [comment]
Given that we use Google Colab for workshops and that Colab only supports Python 3.7, are we hurting ourselves by not providing MDAnalysis for 3.7?

Or are we just waiting for Google to finally upgrade?

**Comment by IAlibay** [comment]
Yeah that's a real pain, the ML workshop had to be delivered using 2.1.0.

There's not much we can do here unfortunately, colab is extremely slow moving, I think it's still using Ubuntu 18.04 as an image. I wouldn't recommend waiting on them, once sklearn, torch et al. have enough non py37 releases they'll be forced to do the update.

**Comment by IAlibay** [comment]
Just to add here, 18.04 drops support by next April, so they'll have to do something by then.

**Comment by orbeckst** [comment]
Ok, we'll just go along with the ecosystem. And tell users to complain to Google ;-).

**Comment by orbeckst** [comment]
Perhaps it might be worth mentioning in the blog post the implications of NEP29 for using Google Colab.

**Comment by IAlibay** [comment]
Sounds good, I think we can probably keep 3.8 around a bit longer, in reality we usually don't drop until ~ 6+ months after the NEP deadline anyways, purely because the latest version of Python always takes forever to roll out upstream. I think we're be a bit more eager to drop numpy versions than working Python ones

**Comment by orbeckst** [comment]
@IAlibay is this blog post still something we need to do? Or should we just roll this into the pre-amble of #198 ?

**Comment by orbeckst** [comment]
I am closing... we've been doing this for a while now.
